src/azul/indexer/__init__.py

```diff
@@ -453,8 +435,22 @@ def reject_joiner(self):
         """
         Raise a requirement error if the given string is found in the bundle
         """
-        self._reject_joiner(self.manifest)
-        self._reject_joiner(self.metadata_files)
+        self._reject_joiner(self.to_json())
```

Comment:
This is too expensive in the AnVIL case. We need to redesign _reject_joiner. I have an idea on how to improve it using `yield` and `JSONEncoder.iterencode`. Luckily we don't need this immediately on AnVIL because the use of the joiner was an HCA convention. In other words we should override this method in the AnVIL subclass to do nothing and add a FIXME "Optimize joiner rejection and re-enable it for AnVIL"

Reply:
https://github.com/DataBiosphere/azul/issues/5256